src/libs/ReportUtils.js

```diff
@@ -1587,32 +1587,41 @@ function getReportIDFromLink(url) {
     return reportID;
 }
 
+/**
+ * @param {Object} report
```

Comment:
Should we add some plain Engligh explanation here so external people can understand this better too?

Reply:
I think that the function name is pretty self-explanatory, but I'm open to suggestions

Reply:
What I meant its more about the business logic underneath which is hidden for the external people so it might be useful to explain when the request money is available for the users in policy expense chats

```suggestion
/**
 * Users can request money in policy expense chats only if they are in a role of a member in the chat (in other words its their policy expense chat)
 * @param {Object} report
```

Reply:
Added the comment